jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -252,3 +253,25 @@ def draw_compass_mpl(image, orig_shape=None, wcs=None, show=True, zoom_limits=No
     plt.close()
 
     return base64.b64encode(buff.getvalue()).decode('utf-8')
+
+
+def data_within_gwcs_bounding_box(data, x, y):
+    """This is for internal use by Imviz coordinates transformation only."""
+    within_bounding_box = True
+    if hasattr(data.coords, '_orig_bounding_box'):
+        # then coords is a GWCS object and had its bounding box cleared
+        # by the Imviz parser
+        ints = data.coords._orig_bounding_box.intervals
+        if isinstance(ints[0].lower, u.Quantity):
+            bb_xmin = ints[0].lower.value
+            bb_xmax = ints[0].upper.value
+            bb_ymin = ints[1].lower.value
+            bb_ymax = ints[1].upper.value
+        else:  # pragma: no cover

```

Comment:
The GWCS official example that I follow in tests demands Quantity, but the actual JWST data is not Quantity. 🤷